packages/run-protocol/package.json

```diff
@@ -10,7 +10,10 @@
   "scripts": {
     "build": "yarn build:bundles",
```

Comment:
Should this build * like 'lint' does?

Reply:
perhaps... but some of the build:* things involve blockchain access, so they don't fit the same workflow.